### ISSUE TITLE
is gov.uk a use case that should be part of first-party sets?

### ISSUE THREAD

**Issue by npdoty** [WICG/first-party-sets#102]
gov.uk is sometimes described in this proposal as an example of a need for first-party sets, because .gov.uk is on the public suffix list, but consent management UI might be the same across different sub-domains and users shouldn't have to repeatedly confirm.

However, the [www.gov.uk privacy notice](https://www.gov.uk/help/privacy-notice) indicates that different subdomains of gov.uk have different data controllers and different privacy policies and are operated by different entities. The cookie consent notices on different subdomains aren't asking the same question at all, but are specific to each agency or department or municipal government.

Can someone clarify what the use case is for domains on the public suffix list that have different registrable domains but where cookies would need to be shared by an embedded party across those separate registrable domains?

**Comment by michael-oneill** [comment]
Hi Nick,

 

One use case I have seen is where a single controller manages multiple country specific domains, which may share a common language e.g. ecample.de, example.at or example.lu, and use them in a unified web experience

 

It then may be a requirement to propagate a low-entropy consent status across the origins e.g. a banner has been seen and so does not need to appear on subsequent pages within the same “session”.

 

Mike

 

 

From: Nick Doty ***@***.***> 
Sent: 18 August 2022 01:31
To: WICG/first-party-sets ***@***.***>
Cc: Subscribed ***@***.***>
Subject: [WICG/first-party-sets] is gov.uk a use case that should be part of first-party sets? (Issue #102)

 

gov.uk is sometimes described in this proposal as an example of a need for first-party sets, because .gov.uk is on the public suffix list, but consent management UI might be the same across different sub-domains and users shouldn't have to repeatedly confirm.

However, the www.gov.uk privacy notice <https://www.gov.uk/help/privacy-notice>  indicates that different subdomains of gov.uk have different data controllers and different privacy policies and are operated by different entities. The cookie consent notices on different subdomains aren't asking the same question at all, but are specific to each agency or department or municipal government.

Can someone clarify what the use case is for domains on the public suffix list that have different registrable domains but where cookies would need to be shared by an embedded party across those separate registrable domains?

**Comment by npdoty** [comment]
> One use case I have seen is where a single controller manages multiple country specific domains, which may share a common language e.g. ecample.de, example.at or example.lu, and use them in a unified web experience

Yes, that's listed as a separate category in the use cases ("Country-specific domains to enable localization"), although I'm not sure how often that should apply either, as I think it would be common for privacy or other practices to also be localized (which is likely why there are using different domains in the first place). If the practices are not the same, I'm not sure why the user would benefit from sharing data between them or automatically applying a single saved choice to all the domains.
